content/5-shape-elements/default.txt

```diff
@@ -511,7 +511,7 @@ text.length()
 
 `returns` __`itself`__
 
-A convenience method to add font-related properties:
```

Comment:
As you said, its a thing :). Would be easier for me when you just change it back :)

Reply:
Great! Sorry about that one. I reverted the commit :)